ui/plugins/pinboard.js

```diff
@@ -101,6 +101,15 @@ treeherder.controller('PinboardCtrl', [
             }
         };
 
+        $scope.retriggerAllPinnedJobs = function() {
+            var jobs = [];
+            // pushing pinned jobs to a list.
+            for (var job in $scope.pinnedJobs) {
+                jobs.push($scope.pinnedJobs[job]);
+            }
+            $scope.retriggerJob(jobs);
```

Comment:
Heh, that the context I was missing. pinnedJobs should probably not be a list, but that's for another bug :smile:
If you want to save a few lines of code feel free to use 
```js
_.values($scope.pinnedJobs)
```